crates/nu-std/lib/log.nu

```diff
@@ -1,49 +1,138 @@
-export def "log CRITICAL_LEVEL" [] { 50 }
-export def "log ERROR_LEVEL"    [] { 40 }
-export def "log WARNING_LEVEL"  [] { 30 }
-export def "log INFO_LEVEL"     [] { 20 }
-export def "log DEBUG_LEVEL"    [] { 10 }
-
-export def "log CRITICAL_LEVEL_PREFIX" [] { "CRT" }
-export def "log ERROR_LEVEL_PREFIX"    [] { "ERR" }
-export def "log WARNING_LEVEL_PREFIX"  [] { "WRN" }
-export def "log INFO_LEVEL_PREFIX"     [] { "INF" }
-export def "log DEBUG_LEVEL_PREFIX"    [] { "DBG" }   
-
-def parse-string-level [level: string] {
+export def "log CRITICAL_LEVEL" [] {
+	50
+}
+
+export def "log ERROR_LEVEL" [] {
+	40
+}
+
+export def "log WARNING_LEVEL" [] {
+	30
+}
+
+export def "log INFO_LEVEL" [] {
+	20
+}
+
+export def "log DEBUG_LEVEL" [] {
+	10
+}
+
+def parse-string-level [
+    level: string
+] {
     (
-        if ($level in [(log CRITICAL_LEVEL_PREFIX) "CRIT" "CRITICAL"]) {
```

Comment:
I don't know if you did it for some reason or it was just `git merge` but I really liked it for keeping the code short and brief

Reply:
moreover this changes the behaviour of the `if`, doesn't it?

Reply:
at least with the `(log ..._LEVEL_PREFIX)` part of the `in [...]`

Reply:
True, the prefixes were lost

Reply:
I've extended conditions with `--short` flags.